zoobc/zoobc-core
# Update max message length and MinRollbackBlocks

## Description
Updating some constants as for new specs in preparation of Mainnet launch.

## Breakdown
- [x] Increased transaction's max message length to:
    - 64KB for non escrow tx
    - 128KB for escrow tx 
- [x] Increased MinRollbackBlocks to 1440, which is on average every half day
- [x] Increased MainchainSnapshotInterval to 1440 to be consistent with previous logic (one snapshot per rollback height)

## Reference Issue
Close #1435 
Close #1436

## Step to Test (optional)
- run the node software for a while and check that first snapshot is taken at height 1440
